pom.xml

```diff
@@ -18,34 +18,17 @@
   <properties>
     <java.version>1.8</java.version>
   </properties>
-
+  <!-- TODO (senyer) Fix jar package dependency conflict causing logs to fail to print -->
   <dependencies>
     <dependency>
       <groupId>com.distkv</groupId>
       <artifactId>distkv-client</artifactId>
-      <version>0.1.3</version>
-      <exclusions>
-        <exclusion>
-          <groupId>org.dousi</groupId>
-          <artifactId>dousi-all</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>com.google.guava</groupId>
-          <artifactId>guava</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>com.distkv</groupId>
-          <artifactId>distkv-rpc</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>com.distkv</groupId>
-          <artifactId>distkv-common</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>ch.qos.logback</groupId>
-          <artifactId>logback-classic</artifactId>
-        </exclusion>
-      </exclusions>
+      <version>0.1.4-SNAPSHOT</version>
```

Comment:
Why change this version?

Reply:
Beacuse  0.1.3 will throws a exception: Class not found: org/dousi/ClientConfig.